### PR TITLE
update new msg and solve some issues

### PR DESCRIPTION
**Pull request by yahyamohmuedpro99**

### CODE REVIEW

**Comment by metmirr** [comment]
`ibc.ts` file line 27, the parameter to createTransaction should be exactly same with the message name in the proto file.

File: src/mappings/ibc.ts [hunk 1]
```diff
@@ -23,5 +23,8 @@ export async function handleMsgRecvPacket(msg:MsgRecvPacketMessage) {
 
 }
 
-
+export async function handleMsgAcknowledgement(msg:MsgAcknowledgementMessage):Promise<void> {
+  const transaction = createTransaction('MsgAcknowledgementMessage',msg)
```
Comment (metmirr) [inline src/mappings/ibc.ts:27]: Parameter this `createTransaction` function must be same with the original message name in this case its `MsgAcknowledgement`.

Comment (yahyamohmuedpro99) [reply src/mappings/ibc.ts:27]: what about others `createTransaction`  need same edit now ?

**Review by metmirr — APPROVED** [review]
Awesome job!

**Pull request merged**